Commands/Commands/utils.js

```diff
@@ -87,6 +87,8 @@ commands.stats = {
           },
           fields: field
         }).catch(bugsnag.notify) // Send Message to DM error
+      }).then(successmsg => {
+        setTimeout(() => bot.Messages.deleteMessages([msg]), config.timeouts.messageDelete)

```

Comment:
Can't bulk-delete a single message.
If this is the `!stats` message, just delete it instantly.